Libraries/config/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+add_library(config SHARED
+            Sources/Config.cpp
```

Comment:
I would, but just because it's extra code. Maybe at some point it'll make sense to pull it out... If it's in `xcsdk`, could even call it something like `xcsdk::AdditionalPaths` to make it really separate from general config.